linkerd/docs/retries.md

```diff
@@ -15,21 +15,36 @@ routers:
         maxMs: 10000
 ```
 
-linkerd can automatically retry requests on certain failures (for example,
-connection errors) and can be configured via the retries block.
+linkerd can automatically retry requests on certain failures and can be
+configured via the retries block.  Retries fall into two categories: retries
+and requeues.
 
 Key | Default Value | Description
 --- | ------------- | -----------
 budget | See [retry budget](#retry-budget-parameters) | Object that determins _how many_ failed requests are eligible to be retried.
 backoff | See [retry backoff](#retry-backoff-parameters) | Object that determines which backoff algorithm should be used.
 
+## Retries
+
+Retries are for application-level failures (such as 5XX responses in the case of HTTP) as determined by the response
+classifier.  If the response classifier determines that a request is a retryable failure, and there is retry budget
+available, then the request will be retried.  Retries are configured by the `retries` parameter on the `service` object.
+On the `retries` object you may specify the retry budget and retry backoff schedule.  Each service has its own retry
+budget that is not shared with other services or clients.
```

Comment:
For here and the rest of the docs changes, can you wrap these lines at 80 charactoers?